tensorboard/backend/application.py

```diff
@@ -282,14 +284,14 @@ def __init__(self, plugins, path_prefix=''):
                              'trailing wildcards are supported '
                              '(i.e., `/.../*`)' %
                              (plugin.plugin_name, path))
-          unordered_prefix_routes[path] = app
+          unordered_prefix_routes[path] = _handling_errors(app)
         else:
           if '*' in path:
             raise ValueError('Plugin %r handles invalid route %r: Only '
                              'trailing wildcards are supported '
                              '(i.e., `/.../*`)' %
                              (plugin.plugin_name, path))
-          self.exact_routes[path] = app
+          self.exact_routes[path] = _handling_errors(app)
```

Comment:
Any particular reason to wrap each route rather than just wrapping `__call__` itself below?  Then it could also be applied in just one place.

Reply:
Good point. I put it here to avoid wrapping _within_ `__call__` on every
request, but you’re absolutely right that we can just wrap `__call__`
itself.

Reply:
Yeah I assume we can just decorate `__call__` itself?  Or we can explicitly wrap the middleware (like above) around the current call implementation, and then cache that and invoke it from `__call__` which amounts to essentially the same thing.

Reply:
LGTM